cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -28,26 +28,280 @@ Cask Data Application Platform Release Notes
+
+- `CDAP-5297 <https://issues.cask.co/browse/CDAP-5297>`__ - Fixed a problem with the CDAP
+  UI not supporting pipelines created using previous versions of Cask Hydrator. The UI now
+  shows appropriate information to upgrade the pipeline to be able to view it in the UI.
+
+- `CDAP-5417 <https://issues.cask.co/browse/CDAP-5417>`__ - Fixed an issue with running
+  the CDAP examples in the CDAP SDK under Windows by setting appropriate memory requirements
+  in the ``cdap.bat`` start script.
+
+- `CDAP-5460 <https://issues.cask.co/browse/CDAP-5460>`__ - Fixed a problem with the
+  workflow	Spark programs status not being updated in the CDAP UI on the program list
+  screen when it is run as a part of Workflow.
+
+- `CDAP-5463 <https://issues.cask.co/browse/CDAP-5463>`__ - Fixed an issue when changing
+  the number of instances of a worker or service.
+
+- `CDAP-5465 <https://issues.cask.co/browse/CDAP-5465>`__ - Fixed a problem |---| in the Cask
+  Hydrator Studio |---| when cloning a batch pipeline with "Spark" as its underlying engine being
+  changed to using a "MapReduce" engine when the clone was completed. Fixed by picking an
+  appropriate engine during the clone process in the CDAP UI.
+
+- `CDAP-5513 <https://issues.cask.co/browse/CDAP-5513>`__ - Fixed a problem with the
+  update of metadata indexes so that search results reflect metadata updates correctly.
+
+- `CDAP-5550 <https://issues.cask.co/browse/CDAP-5550>`__ - Fixed a problem with the
+  workflow statistics HTTP RESTful endpoint. The endpoint now has a default limit of 10 and
+  a default interval of 10 seconds.
+
+- `CDAP-5557 <https://issues.cask.co/browse/CDAP-5557>`__ - Fixed a problem of not showing
+  an appropriate error message in the node configuration when the CDAP backend returns 404
+  for a plugin property.
+
+- `CDAP-5583 <https://issues.cask.co/browse/CDAP-5583>`__ - Added the ability to support
+  multiple sources in the CDAP UI while constructing a pipeline.
+
+- `CDAP-5619 <https://issues.cask.co/browse/CDAP-5619>`__ - Fixed a problem with the
+  importation of a pipeline configuration. If the imported pipeline config doesn't have
+  artifact information for a plugin, the CDAP UI now defaults to the latest artifact from
+  the list of artifacts sent by the backend.
+
+- `CDAP-5629 <https://issues.cask.co/browse/CDAP-5629>`__ - Fixed a problem with losing
+  metadata after changing the stream format on a MapR cluster by avoiding the use of Hive
+  keywords in the CLF format field names; the 'date' field was renamed to 'request_time'.
+
+- `CDAP-5634 <https://issues.cask.co/browse/CDAP-5634>`__ - Fixed a performance issue when
+  rendering/scrolling through large input or output schemas for a plugin in the CDAP UI.
+
+- `CDAP-5652 <https://issues.cask.co/browse/CDAP-5652>`__ - Added command line interface
+  command to retrieve the workflow node states.
+
+- `CDAP-5662 <https://issues.cask.co/browse/CDAP-5662>`__ - Fixed a problem with the CDAP
```

Comment:
affects version is 3.4.0.
@ajainarayanan Does this also affect 3.3.2 or prior?
If not, this doesn't need to be here (or have a release note, right?).